noir_stdlib/src/sha512.nr

```diff
@@ -67,17 +69,21 @@ fn sha_c(msg: [u64; 16], hash: [u64; 8]) -> [u64; 8]
     let w = sha_w(msg);
     for j in 0..80
     {
-        let t1 = out_h[7] + sha_bigma1(out_h[4]) + sha_ch(out_h[4], out_h[5], out_h[6])
-            + K[j] + w[j];
-        let t2 = sha_bigma0(out_h[0]) + sha_maj(out_h[0], out_h[1], out_h[2]);
+                
+        let t1 = crate::wrapping_add(
+                    crate::wrapping_add(
+                        crate::wrapping_add(
+                            crate::wrapping_add(out_h[7] , sha_bigma1(out_h[4])) , sha_ch(out_h[4], out_h[5], out_h[6])),
+                                K[j]) ,w[j]);
```

Comment:
Not a blocker -- can you open a separate PR to remove this nesting and just assign temporary values

Reply:
It's easier to fix this than to create an additional PR :)